src/cmdlib.sh

```diff
@@ -288,6 +287,47 @@ commit_overlay() {
         --skip-list <(echo /statoverride)
 }
 
+create_content_manifest(){
+    mkdir -p "${workdir}"/tmp/buildinfo
+    rpm-ostree compose tree --repo="${tmprepo}" --print-only "${manifest}" > "${manifest_tmp_json}"
+    base_repos=$(jq .repos "${manifest_tmp_json}")
+    rm -f "${manifest_tmp_json}"
+
+    # Get the data form content_sets.yaml and map the repo names given in '$base_repos' to their corresponding
+    # pulp repository IDs provided in https://www.redhat.com/security/data/metrics/repository-to-cpe.json
+    python3 -c "
+import json, yaml;
+# Open the yaml and load the data
+f = open('$1')
+data = yaml.safe_load(f);
+f.close();
+repos=[];
+
+for base_repo in $base_repos:
+    if base_repo in data['repo_mapping']:
+        if data['repo_mapping'][base_repo]['name'] != '':
+            repo_name = data['repo_mapping'][base_repo]['name'].replace('\$ARCH', '$(arch)');
+            repos.append(repo_name)
+        else:
+            print('Warning: No corresponding repo in repository-to-cpe.json for ' + base_repo)
+    else:
+        # Warning message for repositories with no entry in content_sets.yaml
+        print('Warning: No corresponding entry in content_sets.yaml for ' + base_repo)
```

Comment:
What I meant here is that we explicitly error out, but OK with this too to start. We can always tighten things later.

Actually, this is probably better for the developer case. So it's a good default and then we could add a switch for more stricter handling in production pipelines, a bit like how it is for lockfiles.

Reply:
Keeping as it for now.